core/src/main/java/ai/timefold/solver/core/impl/domain/solution/descriptor/SolutionDescriptor.java

```diff
@@ -877,6 +883,10 @@ public PlanningSolutionMetaModel<Solution_> getMetaModel() {
         return planningSolutionMetaModel;
     }
 
+    public boolean hasBothBasicAndListVariables() {
+        return getMetaModel().hasBothBasicAndListVariables();
```

Comment:
Careful with the metamodel. It's supposed to be a simple interface for users - not all methods should go there. Only those that the user would actually need. Arguably, this is not one of them.